server/models/index.ts

```diff
@@ -12,7 +12,9 @@ if (
   !process.env.DATABASE_URL
 ) {
   /* For CI testing */
-  sequelize = new Sequelize('sqlite::memory');
+  sequelize = new Sequelize('sqlite::memory', {
+    logging: false,
```

Comment:
```suggestion
```

It's useful to see what's going on.

Reply:
I was supposed to revert this commit after but I forgot to